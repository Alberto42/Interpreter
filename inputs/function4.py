 y = 0
x = 1
def func(x) {
    y=x
    z = 5

}

z = 6
func(3)