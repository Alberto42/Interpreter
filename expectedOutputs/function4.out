Right (OK,
Ident "z" = VInt 6
Ident "y" = VInt 3
Ident "x" = VInt 1
)
